datasets/tests/management/commands/test_search_vector.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+
+import pytest
+from django.test import TestCase
+
+from datasets.management.commands.searchvector import Command
+from datasets.models import Gazette
+
+
+class TestCommandHandler(TestCase):
+    @pytest.fixture(autouse=True)
+    def capsys(self, capsys):
+        self.capsys = capsys
+
+    @patch.object(Gazette.objects, "update")
+    def test_handler(self, update):
+        command = Command()
+        command.handle()
+
+        captured = self.capsys.readouterr()
+        self.assertRegex(captured.out, r"Creating search vector.*Total items: 0")
+        self.assertRegex(captured.out, r"Done")
```

Comment:
Para casos com regex, você teria que usar o módulo `re` do Python mesmo (embora seja possível criar algo pro `pytest` também: https://kalnytskyi.com/howto/assert-str-matches-regex-in-pytest/).